test/tests/vectorpostprocessors/element_value_sampler/tests

```diff
@@ -0,0 +1,15 @@
+[Tests]
+  [./element_value_sampler]
+    type = 'CSVDiff'
+    input = 'element_value_sampler.i'
+    csvdiff = 'element_value_sampler_element_value_sampler_0000.csv'
+    mesh_mode = REPLICATED
```

Comment:
I'd be nice if this worked with DistributedMesh, but I know the underlying class needs to be updated so I guess this is OK for now.

Reply:
Yes, that will be really nice. Otherwise we will have to use paraview for large problems.